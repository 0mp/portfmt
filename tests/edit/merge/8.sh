echo "PLIST_FILES=" | ${PORTEDIT} merge 8.in | diff -L 8.expected -L 8.actual -u 8.expected -
