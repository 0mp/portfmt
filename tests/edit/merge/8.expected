PORTNAME=	cloak

PLIST_FILES=
